beacon/sync/src/test/java/tech/pegasys/teku/beacon/sync/historical/HistoricalBlockSyncServiceTest.java

```diff
@@ -70,8 +71,13 @@ public class HistoricalBlockSyncServiceTest {
 
   private final SyncStateProvider syncStateProvider = mock(SyncStateProvider.class);
 
+  private final CombinedChainDataClient chainData = mock(CombinedChainDataClient.class);
+  private final Optional<String> genesisStateResource =
+      Optional.of(
+          "https://2driVVdXx:5058c7@eth2-beacon-mainnet.infura.io/eth/v1/debug/beacon/states/finalized");
```

Comment:
I probably wouldn't use an infura URL here (and if that's a real access key you need to revoke access to that project now).  Just a `https://example.com/state.ssz` type of URL is generally best for testing so there's no risk of the test actually hitting a real web server (example.com is specifically reserved for examples so can't be registered).